### ISSUE TITLE
Support for not using ShadowDOM and guidance on contributing new hooks

### ISSUE THREAD

**Issue by jpray** [matthewp/haunted#56]
This library is my favorite thing since lit-html/hyperHTML and a joy to work with.  Major kudos for creating it.  I have 2 questions:

1) I don't use ShadowDOM.  It was easy adding an option to make this optional (https://github.com/jpray/haunted/commit/044da1ed744284c38eddd646befe9589197f29dc).  If I add tests and documentation, is this something you'd consider?

2) I've created a number of hooks, some of which might be useful to others.  Do you see the number of hooks added to this project continuing to grow?  If so, would the idea be that unused hooks in an app should be tree-shaken away during a build (since they are all being imported and re-exported from the base module).  Or should they exist in the library but require a separate import? e.g:
```
import { component } from "haunted";
import { useSharedState } from "haunted/hooks/use-shared-state";
```
I guess another alternative would be for a "haunted-extra" library to popup that has lots more hooks in it.  What would you like to see happen?

**Comment by jpray** [comment]
Chewing on #2 a bit more, I'm wondering if this project could benefit from a mono repo approach and the @haunted npm scope.  Then lots of hooks could be published under @haunted/<hookname>., and probably all of the hooks in the base package could be published separately then.

**Comment by matthewp** [comment]
@jpray I would absolutely accept a PR! As for docs, I assume you want this for IE11 support. There's currently some stuff in the Use section about IE11: https://github.com/matthewp/haunted#use . Probably if you are going to add some more, adding a new heading for IE11 would be a good idea.

2. My aim is to only include the same hooks that React provides. A haunted-hooks or something would indeed be a good way to go. If someone wants to grab a `@haunted/` or something namespace on npm to share with the community that would be cool too.

Happy to have more contributions to Haunted, thank you!

**Comment by matthewp** [comment]
Fixed by #57
